tensorboard/data/server/reservoir.rs

```diff
@@ -226,57 +278,62 @@ mod tests {
     #[test]
     fn test() {
         let mut rsv = StageReservoir::with_control(7, ScriptedControl::new());
-        let mut head = Vec::new();
+        let mut head = Basin::new();
```

Comment:
Maybe worth renaming the test locals from `head` to `basin` as well, but up to you.

Reply:
Ah, sorry; missed this one. It doesn’t bother me; I’ll keep it as is.